quantuminspire/cli/command_list.py

```diff
@@ -233,19 +233,19 @@ def sync_projects(
 @files_app.command("upload")
 def upload_files(
     name: str = typer.Argument(..., help="The name of the file to upload"),
-    runtime_type_id: int = typer.Argument(
-        ..., help="The id of the runtime type on which this algorithm should be executed"
+    backend_type_id: int = typer.Argument(
+        ..., help="The id of the backend type on which this algorithm should be executed"
     ),
 ) -> None:
     """Upload a file to the QI API.
 
     Upload a Hybrid Quantum/Classical Algorithm to the Quantum Inspire API. This file is marked as a hybrid algorithm
     when sent to the API.
     """
-    runtime = RemoteRuntime()
+    backend = RemoteBackend()
     program = HybridAlgorithm(platform_name="spin-2", program_name=name)
     program.read_file(Path(name))
-    run_id = runtime.run(program, runtime_type_id=runtime_type_id)
+    run_id = backend.run(program, backend_type_id=backend_type_id)
     typer.echo(f"Upload file with name: {name}")
     typer.echo(f"run_id {run_id}")
```

Comment:
I think run_id should also be replaced here, since this is also printed as info for the end user.

Reply:
Strange, did a global find replace but still missed a few, updated